src/ziplist.h

```diff
@@ -34,6 +34,12 @@
 #define ZIPLIST_HEAD 0
 #define ZIPLIST_TAIL 1
 
+typedef struct {
+    unsigned char *sval;
```

Comment:
It would be nice to document this just to say that if sval == NULL, then it's a long long value.